### PR TITLE
feat(admin-ui): UMA Resources Information

### PR DESCRIPTION
**Pull request by harryandriyan**
- [x] RPT modification script (**Figure -1**)
Shift `UMA RPT Claims` to this page from `Client Scripts` page.
image

- [x] add UMA resource information in oidc clients add/edit pages (**Figure -1**)

**UMA resource Api**: [/jans-config-api/api/v1/uma/resources/clientId/{clientId}](https://gluu.org/swagger-ui/?url=https://raw.githubusercontent.com/JanssenProject/jans/main/jans-config-api/docs/jans-config-api-swagger.yaml#/operations/OAuth%20-%20UMA%20Resources/get-oauth-uma-resources-by-clientid)

**Field Mapping**
Show `id`, `name` from api. 
- [x] on click of resource `id` link it should open page providing resource details. Here, we also need to show scope details. (**Figure -2**)

**scope Api**: /jans-config-api/api/v1/scopes/{inum}

### CODE REVIEW

File: admin-ui/plugins/auth-server/components/Clients/ClientUMADetailPage.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  umaResources,
+}) {
+  const { t } = useTranslation()
+  const { id } = useParams()
+  const [selectedUMA, setSelectedUMA] = useState()
+  const [scopeExpression, setScopeExpression] = useState()
+
+  useEffect(() => {
+    if (isEmpty(umaResources)) {
+      dispatch(getUMAResourcesByClient(clientData?.inum))
```
Comment (duttarnab) [inline admin-ui/plugins/auth-server/components/Clients/ClientUMADetailPage.js:32]: why again fetching UMA resources by client here? We can just pass it from `ClientCibaParUmaPanel.js`.

File: admin-ui/plugins/auth-server/redux/api/OIDCApi.js [hunk 2]
```diff
@@ -35,6 +35,14 @@ export default class OIDCApi {
     })
   }
 
+  getUMAResources = async (clientId) => {
```
Comment (duttarnab) [inline admin-ui/plugins/auth-server/redux/api/OIDCApi.js:38]: can we have this in separate UMAResourceApi.js file.

File: admin-ui/plugins/auth-server/redux/reducers/OIDCReducer.js [hunk 3]
```diff
@@ -127,7 +130,19 @@ export default function oidcReducer(state = INIT_STATE, action) {
       } else {
         return handleDefault()
       }
+    case GET_UMA_RESOURCES:
```
Comment (duttarnab) [inline admin-ui/plugins/auth-server/redux/reducers/OIDCReducer.js:133]: please create a separate reducer for uma resource.

File: admin-ui/plugins/auth-server/redux/sagas/OIDCSaga.js [hunk 4]
```diff
@@ -45,6 +47,21 @@ function* newFunction() {
   return new OIDCApi(api)
 }
 
+function* newUMAFunction() {
```
Comment (duttarnab) [inline admin-ui/plugins/auth-server/redux/sagas/OIDCSaga.js:50]: please create a separate saga for uma resource.

**Review by duttarnab — CHANGES_REQUESTED** [review]

File: admin-ui/plugins/auth-server/redux/sagas/OIDCSaga.js (hunk 4 above)
Comment (harryandriyan) [reply admin-ui/plugins/auth-server/redux/sagas/OIDCSaga.js]: done, updated @duttarnab

File: admin-ui/plugins/auth-server/redux/reducers/OIDCReducer.js (hunk 3 above)
Comment (harryandriyan) [reply admin-ui/plugins/auth-server/redux/reducers/OIDCReducer.js]: done, updated @duttarnab

File: admin-ui/plugins/auth-server/redux/api/OIDCApi.js (hunk 2 above)
Comment (harryandriyan) [reply admin-ui/plugins/auth-server/redux/api/OIDCApi.js]: done, updated @duttarnab

File: admin-ui/plugins/auth-server/components/Clients/ClientUMADetailPage.js (hunk 1 above)
Comment (harryandriyan) [reply admin-ui/plugins/auth-server/components/Clients/ClientUMADetailPage.js]: done, updated @duttarnab

**Review by duttarnab — APPROVED** [review]

**Review by duttarnab — APPROVED** [review]

**Pull request merged**